contracts/world/sources/assemblies/assembly.move

```diff
@@ -27,6 +27,7 @@ public struct AssemblyRegistry has key {
 
 public struct Assembly has key {
     id: UID,
+    owner_character_id: ID,
```

Comment:
If an assembly changes hands, i.e. in the case of an ownership transfer. will we need to update this field? 

Asking because if we add this field it seems to me that a regular Sui transfer of the OwnerCap object will not be possible unless our systems update this field.

Reply:
Yes agreed, Is it okay If I create a new PR to remove character ID coupling from all the assemblies ?

Reply:
Sure, will this PR then be updated so the field being introduced is explicitly an `owner_cap_id`?